ansible_collections/serverscom/sc_api/plugins/modules/sc_dedicated_server_power.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2020, Servers.com
+# GNU General Public License v3.0
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import absolute_import, division, print_function
+
+
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = """
+---
+module: sc_dedicated_server_power
+version_added: "1.0.0"
+author: "Volodymyr Rudniev (@koef)"
+short_description: Power on/off Bare Metal server
+description: >
+    Manage the power state of a Bare Metal server outlet. Power on will start
+    the server only if OOB configured accordingly.
+
+options:
+  endpoint:
+    type: str
+    default: "https://api.servers.com/v1"
+    description:
+      - API endpoint to connect to.
+
+  token:
+    type: str
+    required: true
+    description:
+      - API bearer token.
+      - Obtain from https://portal.servers.com under Profile → Public API.
+
+  server_id:
+    type: str
+    required: true
+    description:
+      - ID of the Bare Metal server.
+      - Use I(serverscom.sc_api.sc_baremetal_servers_info) to retrieve servers.
+
+  state:
+    type: str
+    required: true
+    choices: ['on', 'off', 'cycle']
+    description:
+      - Desired power state.
+      - "I(on): power on outlet."
+      - "I(off): power off outlet."
+      - "I(cycle): power cycle outlet."
```

Comment:
I think, we should explain what 'cycle' means. We got used to it, but for a user it may be a bit hard to decipher.